api/v1alpha1/etcdcluster_types.go

```diff
@@ -9,6 +9,8 @@ import (
 
 // EtcdClusterSpec defines the desired state of EtcdCluster
 type EtcdClusterSpec struct {
+	// Number of instances of etcd to assemble into this cluster
+	Replicas *int32 `json:"replicas"`
```

Comment:
There's a new way to annotate this with a default value: 
 * https://github.com/kubernetes-sigs/controller-tools/pull/323

But it's only just merged. We could add the annotation anyway, I suppose. What do you think?

Reply:
If it's only merged 21 days ago probably best to leave it for now.

> We could add the annotation anyway, I suppose.

I'd be concerned that a future upgrade of the controller-tools could suddenly change this behaviour in that case. I'd rather we add this on purpose in the future.

That, or we upgrade controller-tools now to get the new annotation.

Reply:
Agreed